nox.py

```diff
@@ -320,3 +320,12 @@ def mark_if_necessary(requirement):
         for requirement in sorted(requirements, key=lambda s: s.lower()):
             if not requirement.startswith('#'):
                 f.write(requirement)
+
+
+def session_readmegen(session):
+    session.install('-r', 'testing/requirements-dev.txt')
+
+    in_files = list(list_files('.', 'README.rst.in'))
```

Comment:
I think `.in` is clearer that it's used exclusively for generating the `README.rst` file, but I'm happy to be disagreed with. @waprin, thoughts?